src/views/List.jsx

```diff
@@ -1,18 +1,46 @@
+import { useState } from 'react';
 import { ListItem } from '../components';
 
 export function List({ data }) {
+	const [filter, setFilter] = useState('');
+
+	const clickHandler = (event) => {
+		event.preventDefault();
+		setFilter('');
+	};
+
 	return (
 		<>
-			<p>
-				Hello from the <code>/list</code> page!
-			</p>
+			<form>
+				<label htmlFor="itemInput">Filter Items: </label>
+				<input
+					type="text"
+					id="itemInput"
+					name="itemInput"
+					placeholder="Start typing here..."
+					value={filter}
+					onChange={(event) => setFilter(event.target.value)}
+				/>
+				{filter && (
+					<span>
+						<button type="reset" onClick={clickHandler}>
+							X
+						</button>
+					</span>
+				)}
+			</form>
 			<ul>
 				{data.length > 0 ? (
-					<ul>
-						{data.map((item) => {
-							return <ListItem name={item.name} key={item.id} />;
-						})}
-					</ul>
+					<li>
```

Comment:
I'm finding that list items are being nested inside of another list item, I think this `li` can be removed? <img width="527" alt="Screenshot 2023-01-24 at 3 06 27 PM" src="https://user-images.githubusercontent.com/1530227/214440981-47a552f5-c35a-4c10-afbc-5907fa32f203.png">

Reply:
Whoops! Yes I had to just move the position of the <ul> tags and then I was able to get rid of the <li> tags of the List.jsx file. Thanks for catching that!

Reply:
Whoops, I didn't look deeper into the component within the map, just saw the outer tags. Great catch @luisaugusto, thanks!